OSPatching/README.md

```diff
@@ -337,46 +307,44 @@ You can stop the OS updates for debugging. Once the “stop” parameter is set
     "local": false,
     "idleTestScript": "<path_to_idletestscript>",
     "healthyTestScript": "<path_to_healthytestscript>"
-  }
-}
-```
-If the `vmStatusTest` scripts are stored in Azure Storage private containers, you have to provide the `storageAccountName` and `storageAccountKey`.
-* Protected Settings
-```json
-{
+  },
   "storageAccountName": "MyAccount",
   "storageAccountKey": "Mykey"
 }
 ```
 
-### 3.5 Enable the extension repeatly
-Enabling the extension with the exactly same configurations is unaccepted in current design.
-If you need to run scripts repeatly, you can add a timestamp.
+### 3.5 Enable the extension repeatedly
+Enabling the OSPatching Extension with the exact same configuration is unsupported and will result in
+a no-op (nothing will happen). If you need to run scripts repeatedly, you can add a timestamp.
 
 ```json
 "timestamp": 123456789
 ```
 
 ### 3.6 Disable the extension
-If you want to switch to manual OS update temporarily, you can set the `disable` parameter to `true` which won't uninstall the OSPatching extension.
+If you want to switch to manual OS update temporarily, you can set the `disable` parameter to `true` instead of uninstalling the OSPatching Extension.
 
 ## Supported Linux Distributions
 - Ubuntu 12.04 and higher
 - CentOS 6.5 and higher
 - Oracle Linux 6.4.0.0.0 and higher
-- openSUSE 13.1 and higher
+- OpenSUSE 13.1 and higher
 - SUSE Linux Enterprise Server 11 SP3 and higher
 
-## Debug
-
-* The status of the extension is reported back to Azure so that user can
-see the status on Azure Portal
+## Debugging
 * The operation log of the extension is `/var/log/azure/<extension-name>/<version>/extension.log` file.
+* The installation status of the extension is reported back to Azure so that the user can see the status on Azure Portal.
+  This does not mean the OSPatching Extension successfully applied the current configuration to the VM.
+* Attempting to enable the OSPatching Extension 2 or more times with the same configuration will result in nothing happening.
+  See [Enable the extension repeatedly](#3.5 Enable the extension repeatedly) section above for more details.
+
+## Mitigations
+If the OSPatching Extension is not working, your Linux distro likely has a recommended way to configure automatic updates. Some distros such as Ubuntu will have an automated way to keep your machine patched. Please refer to your distro's documentation for more information.
```

Comment:
Is this a recommendation to just use the Distro's preferred method of OS patching?  If so, I agree, but I think the wording could be better.  If you know of a existing process for the endorsed distros I would add that text here.